src/data.h

```diff
@@ -58,6 +64,12 @@ struct PFNeutralObj {
 	etaphi_t hwEta, hwPhi; // relative to the region center, at calo
 	particleid_t hwId;
 };
+struct PFMuonObj {
+	pt_t hwPt;
+	etaphi_t hwEta, hwPhi; // relative to the region center, at vtx(?)
+	particleid_t hwId;
+	z0_t hwZ0;
+};
```

Comment:
at the moment you're right we can re-use `PFChargedObj` but I read this to mean "charged hadron" and created a new object that could diverge later with specific muon information -- maybe isolation?  or it may need different precisions?  I can revert for now if desired.